### ISSUE TITLE
pytest not loading pytest_metadata when installing 2.0.0 from pypi

### ISSUE THREAD

**Issue by eltrufas** [pytest-dev/pytest-metadata#40]
I can reproduce this with
```
#!/bin/bash
set -x

virtualenv venv
source venv/bin/activate
pip install pytest pytest-metadata
pytest --trace-config
```

Output:
```
+ virtualenv venv
created virtual environment CPython3.10.2.final.0-64 in 149ms
  creator CPython3Posix(dest=/tmp/test/venv, clear=False, no_vcs_ignore=False, global=False)
  seeder FromAppData(extra_search_dir=/usr/share/python-wheels,download=False, pip=bundle, setuptools=bundle, wheel=bundle, via=copy, app_data_dir=/home/rcastillo/.local/share/virtualenv)
    added seed packages: pip==21.2.3, setuptools==57.4.0, wheel==0.36.2
  activators BashActivator,CShellActivator,FishActivator,NushellActivator,PowerShellActivator,PythonActivator
+ source venv/bin/activate
++ '[' venv/bin/activate = ./reproduce.sh ']'
++ deactivate nondestructive
++ unset -f pydoc
++ '[' -z '' ']'
++ '[' -z '' ']'
++ hash -r
++ '[' -z '' ']'
++ unset VIRTUAL_ENV
++ '[' '!' nondestructive = nondestructive ']'
++ VIRTUAL_ENV=/tmp/test/venv
++ '[' linux-gnu = cygwin ']'
++ '[' linux-gnu = msys ']'
++ export VIRTUAL_ENV
++ _OLD_VIRTUAL_PATH=/tmp/test/venv/bin:/home/rcastillo/.local/bin:/home/rcastillo/bin:/usr/local/bin:/usr/local/sbin:/usr/bin:/usr/sbin:/home/rcastillo/.fzf/bin:/usr/local/go/bin:/home/rcastillo/go/bin:/usr/local/go/bin:/home/rcastillo/go/bin
++ PATH=/tmp/test/venv/bin:/tmp/test/venv/bin:/home/rcastillo/.local/bin:/home/rcastillo/bin:/usr/local/bin:/usr/local/sbin:/usr/bin:/usr/sbin:/home/rcastillo/.fzf/bin:/usr/local/go/bin:/home/rcastillo/go/bin:/usr/local/go/bin:/home/rcastillo/go/bin
++ export PATH
++ '[' -z '' ']'
++ '[' -z '' ']'
++ _OLD_VIRTUAL_PS1=
++ '[' x '!=' x ']'
+++ basename /tmp/test/venv
++ PS1='(venv) '
++ export PS1
++ alias pydoc
++ true
++ hash -r
+ pip install pytest pytest-metadata
Collecting pytest
  Using cached pytest-7.1.1-py3-none-any.whl (297 kB)
Collecting pytest-metadata
  Using cached pytest_metadata-2.0.0-py3-none-any.whl (7.0 kB)
Collecting iniconfig
  Using cached iniconfig-1.1.1-py2.py3-none-any.whl (5.0 kB)
Collecting packaging
  Using cached packaging-21.3-py3-none-any.whl (40 kB)
Collecting tomli>=1.0.0
  Using cached tomli-2.0.1-py3-none-any.whl (12 kB)
Collecting py>=1.8.2
  Using cached py-1.11.0-py2.py3-none-any.whl (98 kB)
Collecting attrs>=19.2.0
  Using cached attrs-21.4.0-py2.py3-none-any.whl (60 kB)
Collecting pluggy<2.0,>=0.12
  Using cached pluggy-1.0.0-py2.py3-none-any.whl (13 kB)
Collecting pyparsing!=3.0.5,>=2.0.2
  Using cached pyparsing-3.0.7-py3-none-any.whl (98 kB)
Installing collected packages: pyparsing, tomli, py, pluggy, packaging, iniconfig, attrs, pytest, pytest-metadata
Successfully installed attrs-21.4.0 iniconfig-1.1.1 packaging-21.3 pluggy-1.0.0 py-1.11.0 pyparsing-3.0.7 pytest-7.1.1 pytest-metadata-2.0.0 tomli-2.0.1
WARNING: You are using pip version 21.2.3; however, version 22.0.4 is available.
You should consider upgrading via the '/tmp/test/venv/bin/python -m pip install --upgrade pip' command.
+ pytest --trace-config
PLUGIN registered: <_pytest.config.PytestPluginManager object at 0x7f8512864580>
PLUGIN registered: <_pytest.config.Config object at 0x7f8511e4f070>
PLUGIN registered: <module '_pytest.mark' from '/tmp/test/venv/lib/python3.10/site-packages/_pytest/mark/__init__.py'>
PLUGIN registered: <module '_pytest.main' from '/tmp/test/venv/lib/python3.10/site-packages/_pytest/main.py'>
PLUGIN registered: <module '_pytest.runner' from '/tmp/test/venv/lib/python3.10/site-packages/_pytest/runner.py'>
PLUGIN registered: <module '_pytest.fixtures' from '/tmp/test/venv/lib/python3.10/site-packages/_pytest/fixtures.py'>
PLUGIN registered: <module '_pytest.helpconfig' from '/tmp/test/venv/lib/python3.10/site-packages/_pytest/helpconfig.py'>
PLUGIN registered: <module '_pytest.python' from '/tmp/test/venv/lib/python3.10/site-packages/_pytest/python.py'>
PLUGIN registered: <module '_pytest.terminal' from '/tmp/test/venv/lib/python3.10/site-packages/_pytest/terminal.py'>
PLUGIN registered: <module '_pytest.debugging' from '/tmp/test/venv/lib/python3.10/site-packages/_pytest/debugging.py'>
PLUGIN registered: <module '_pytest.unittest' from '/tmp/test/venv/lib/python3.10/site-packages/_pytest/unittest.py'>
PLUGIN registered: <module '_pytest.capture' from '/tmp/test/venv/lib/python3.10/site-packages/_pytest/capture.py'>
PLUGIN registered: <module '_pytest.skipping' from '/tmp/test/venv/lib/python3.10/site-packages/_pytest/skipping.py'>
PLUGIN registered: <module '_pytest.legacypath' from '/tmp/test/venv/lib/python3.10/site-packages/_pytest/legacypath.py'>
PLUGIN registered: <module '_pytest.tmpdir' from '/tmp/test/venv/lib/python3.10/site-packages/_pytest/tmpdir.py'>
PLUGIN registered: <module '_pytest.monkeypatch' from '/tmp/test/venv/lib/python3.10/site-packages/_pytest/monkeypatch.py'>
PLUGIN registered: <module '_pytest.recwarn' from '/tmp/test/venv/lib/python3.10/site-packages/_pytest/recwarn.py'>
PLUGIN registered: <module '_pytest.pastebin' from '/tmp/test/venv/lib/python3.10/site-packages/_pytest/pastebin.py'>
PLUGIN registered: <module '_pytest.nose' from '/tmp/test/venv/lib/python3.10/site-packages/_pytest/nose.py'>
PLUGIN registered: <module '_pytest.assertion' from '/tmp/test/venv/lib/python3.10/site-packages/_pytest/assertion/__init__.py'>
PLUGIN registered: <module '_pytest.junitxml' from '/tmp/test/venv/lib/python3.10/site-packages/_pytest/junitxml.py'>
PLUGIN registered: <module '_pytest.doctest' from '/tmp/test/venv/lib/python3.10/site-packages/_pytest/doctest.py'>
PLUGIN registered: <module '_pytest.cacheprovider' from '/tmp/test/venv/lib/python3.10/site-packages/_pytest/cacheprovider.py'>
PLUGIN registered: <module '_pytest.freeze_support' from '/tmp/test/venv/lib/python3.10/site-packages/_pytest/freeze_support.py'>
PLUGIN registered: <module '_pytest.setuponly' from '/tmp/test/venv/lib/python3.10/site-packages/_pytest/setuponly.py'>
PLUGIN registered: <module '_pytest.setupplan' from '/tmp/test/venv/lib/python3.10/site-packages/_pytest/setupplan.py'>
PLUGIN registered: <module '_pytest.stepwise' from '/tmp/test/venv/lib/python3.10/site-packages/_pytest/stepwise.py'>
PLUGIN registered: <module '_pytest.warnings' from '/tmp/test/venv/lib/python3.10/site-packages/_pytest/warnings.py'>
PLUGIN registered: <module '_pytest.logging' from '/tmp/test/venv/lib/python3.10/site-packages/_pytest/logging.py'>
PLUGIN registered: <module '_pytest.reports' from '/tmp/test/venv/lib/python3.10/site-packages/_pytest/reports.py'>
PLUGIN registered: <module '_pytest.python_path' from '/tmp/test/venv/lib/python3.10/site-packages/_pytest/python_path.py'>
PLUGIN registered: <module '_pytest.unraisableexception' from '/tmp/test/venv/lib/python3.10/site-packages/_pytest/unraisableexception.py'>
PLUGIN registered: <module '_pytest.threadexception' from '/tmp/test/venv/lib/python3.10/site-packages/_pytest/threadexception.py'>
PLUGIN registered: <module '_pytest.faulthandler' from '/tmp/test/venv/lib/python3.10/site-packages/_pytest/faulthandler.py'>
PLUGIN registered: <CaptureManager _method='fd' _global_capturing=<MultiCapture out=<FDCapture 1 oldfd=5 _state='suspended' tmpfile=<_io.TextIOWrapper name="<_io.FileIO name=6 mode='rb+' closefd=True>" mode='r+' encoding='utf-8'>> err=<FDCapture 2 oldfd=7 _state='suspended' tmpfile=<_io.TextIOWrapper name="<_io.FileIO name=8 mode='rb+' closefd=True>" mode='r+' encoding='utf-8'>> in_=<FDCapture 0 oldfd=3 _state='started' tmpfile=<_io.TextIOWrapper name='/dev/null' mode='r' encoding='UTF-8'>> _state='suspended' _in_suspended=False> _capture_fixture=None>
PLUGIN registered: <Session test exitstatus=<ExitCode.OK: 0> testsfailed=0 testscollected=0>
PLUGIN registered: <_pytest.cacheprovider.LFPlugin object at 0x7f8511cbb220>
PLUGIN registered: <_pytest.cacheprovider.NFPlugin object at 0x7f8511cbab00>
PLUGIN registered: <class '_pytest.legacypath.LegacyTmpdirPlugin'>
PLUGIN registered: <_pytest.terminal.TerminalReporter object at 0x7f8511cbab90>
PLUGIN registered: <_pytest.logging.LoggingPlugin object at 0x7f8511cbaec0>
PLUGIN registered: <_pytest.fixtures.FixtureManager object at 0x7f8511e4f220>
============================= test session starts ==============================
platform linux -- Python 3.10.2, pytest-7.1.1, pluggy-1.0.0
using: pytest-7.1.1
active plugins:
    140209518167424     : <_pytest.config.PytestPluginManager object at 0x7f8512864580>
    pytestconfig        : <_pytest.config.Config object at 0x7f8511e4f070>
    mark                : /tmp/test/venv/lib/python3.10/site-packages/_pytest/mark/__init__.py
    main                : /tmp/test/venv/lib/python3.10/site-packages/_pytest/main.py
    runner              : /tmp/test/venv/lib/python3.10/site-packages/_pytest/runner.py
    fixtures            : /tmp/test/venv/lib/python3.10/site-packages/_pytest/fixtures.py
    helpconfig          : /tmp/test/venv/lib/python3.10/site-packages/_pytest/helpconfig.py
    python              : /tmp/test/venv/lib/python3.10/site-packages/_pytest/python.py
    terminal            : /tmp/test/venv/lib/python3.10/site-packages/_pytest/terminal.py
    debugging           : /tmp/test/venv/lib/python3.10/site-packages/_pytest/debugging.py
    unittest            : /tmp/test/venv/lib/python3.10/site-packages/_pytest/unittest.py
    capture             : /tmp/test/venv/lib/python3.10/site-packages/_pytest/capture.py
    skipping            : /tmp/test/venv/lib/python3.10/site-packages/_pytest/skipping.py
    legacypath          : /tmp/test/venv/lib/python3.10/site-packages/_pytest/legacypath.py
    tmpdir              : /tmp/test/venv/lib/python3.10/site-packages/_pytest/tmpdir.py
    monkeypatch         : /tmp/test/venv/lib/python3.10/site-packages/_pytest/monkeypatch.py
    recwarn             : /tmp/test/venv/lib/python3.10/site-packages/_pytest/recwarn.py
    pastebin            : /tmp/test/venv/lib/python3.10/site-packages/_pytest/pastebin.py
    nose                : /tmp/test/venv/lib/python3.10/site-packages/_pytest/nose.py
    assertion           : /tmp/test/venv/lib/python3.10/site-packages/_pytest/assertion/__init__.py
    junitxml            : /tmp/test/venv/lib/python3.10/site-packages/_pytest/junitxml.py
    doctest             : /tmp/test/venv/lib/python3.10/site-packages/_pytest/doctest.py
    cacheprovider       : /tmp/test/venv/lib/python3.10/site-packages/_pytest/cacheprovider.py
    freeze_support      : /tmp/test/venv/lib/python3.10/site-packages/_pytest/freeze_support.py
    setuponly           : /tmp/test/venv/lib/python3.10/site-packages/_pytest/setuponly.py
    setupplan           : /tmp/test/venv/lib/python3.10/site-packages/_pytest/setupplan.py
    stepwise            : /tmp/test/venv/lib/python3.10/site-packages/_pytest/stepwise.py
    warnings            : /tmp/test/venv/lib/python3.10/site-packages/_pytest/warnings.py
    logging             : /tmp/test/venv/lib/python3.10/site-packages/_pytest/logging.py
    reports             : /tmp/test/venv/lib/python3.10/site-packages/_pytest/reports.py
    python_path         : /tmp/test/venv/lib/python3.10/site-packages/_pytest/python_path.py
    unraisableexception : /tmp/test/venv/lib/python3.10/site-packages/_pytest/unraisableexception.py
    threadexception     : /tmp/test/venv/lib/python3.10/site-packages/_pytest/threadexception.py
    faulthandler        : /tmp/test/venv/lib/python3.10/site-packages/_pytest/faulthandler.py
    capturemanager      : <CaptureManager _method='fd' _global_capturing=<MultiCapture out=<FDCapture 1 oldfd=5 _state='suspended' tmpfile=<_io.TextIOWrapper name="<_io.FileIO name=6 mode='rb+' closefd=True>" mode='r+' encoding='utf-8'>> err=<FDCapture 2 oldfd=7 _state='suspended' tmpfile=<_io.TextIOWrapper name="<_io.FileIO name=8 mode='rb+' closefd=True>" mode='r+' encoding='utf-8'>> in_=<FDCapture 0 oldfd=3 _state='started' tmpfile=<_io.TextIOWrapper name='/dev/null' mode='r' encoding='UTF-8'>> _state='suspended' _in_suspended=False> _capture_fixture=None>
    session             : <Session test exitstatus=<ExitCode.OK: 0> testsfailed=0 testscollected=0>
    lfplugin            : <_pytest.cacheprovider.LFPlugin object at 0x7f8511cbb220>
    nfplugin            : <_pytest.cacheprovider.NFPlugin object at 0x7f8511cbab00>
    legacypath-tmpdir   : <class '_pytest.legacypath.LegacyTmpdirPlugin'>
    terminalreporter    : <_pytest.terminal.TerminalReporter object at 0x7f8511cbab90>
    logging-plugin      : <_pytest.logging.LoggingPlugin object at 0x7f8511cbaec0>
    funcmanage          : <_pytest.fixtures.FixtureManager object at 0x7f8511e4f220>
rootdir: /tmp/test
collected 0 items

============================ no tests ran in 0.00s =============================
```

**Comment by ChrisWard42** [comment]
We observed the same thing earlier today, package installed fine but --metadata wasn't treated as a registered CLI option. Same issue with Pytest not loading it.

**Comment by BeyondEvil** [comment]
Thanks for reporting this!

**Comment by BeyondEvil** [comment]
@eltrufas @ChrisWard42 

Do you mind trying the version I uploaded to test.pypi? 

https://test.pypi.org/project/pytest-metadata/2.0.1rc2/

**Comment by BeyondEvil** [comment]
Should be fixed in 2.0.1. Thanks! 😊
